tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -1349,3 +1349,64 @@ def test_revert_migrated_tables_failed(caplog):
     table_migrate = get_table_migrator(backend)
     table_migrate.revert_migrated_tables(schema="test_schema1")
     assert "Failed to revert table hive_metastore.test_schema1.test_table1: error" in caplog.text
+
+
+def test_refresh_migration_status_published_remained_tables(caplog):
```

Comment:
Not for now: this test needs a lot of setup, maybe it makes sense to move setup code to a fixture IF it can be shared with other tests

Reply:
An issue was created. [#2205](https://github.com/databrickslabs/ucx/pull/2205)